src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -34,6 +34,9 @@ public class ExecutionOptions : IProvideUserContext
         /// <summary>Input variables to GraphQL request</summary>
         public Inputs Inputs { get; set; }
 
+        /// <summary>Extension variables to GraphQL request</summary>
+        public Inputs Extensions { get; set; }
```

Comment:
I have decieded to reuse the Inputs type because Extensions work exactly like Variables.